mkn.yaml

```
@@ -0,0 +1,217 @@
+
+- name: base
+  parent: ${mkn.base}
+
+
+- name: core
+  parent: base
+  inc: src subprojects subprojects/phlop/inc
+  src: src/core subprojects/phlop/src/phlop/timing/threaded_scope_timer.cpp
+  out: phare_core
+  sub:
+      menum&subprojects/menum(https://github.com/Neargye/magic_enum)
```

yamllint

[warning] wrong indentation: expected 4 but found 6

(indentation)